ldsc/src/main/resources/translatePartitionedHeritability.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+import glob
+import json
+import math
+import numpy as np
+import os
+import re
+from scipy.stats import t as tdist
+import subprocess
+
+downloaded_files = '/mnt/var/ldsc'
+s3_path = 's3://dig-analysis-data/out/ldsc/partitioned_heritability'
+
+
+def get_phenotype_annot_ancestries():
+    out = {}
+    for file in glob.glob(f'{downloaded_files}/*/*/*'):
+        ancestry, phenotype, annot = re.findall('.*/(.+).results', file)[0].split('.')
+        if phenotype not in out:
+            out[phenotype] = {}
+        if annot not in out[phenotype]:
+            out[phenotype][annot] = []
+        out[phenotype][annot].append(ancestry)
+    return out
+
+
+def translate(file):
+    with open(file, 'r') as f:
+        # Get last line of file
+        last_line = [a.strip() for a in f.readlines()[-1].strip().split('\t')]
+        return {
+            'snps': float(last_line[1]),
+            'h2': {'beta': float(last_line[2]), 'stdErr': float(last_line[3])},
+            'enrichment': {'beta': float(last_line[4]), 'stdErr': float(last_line[5])},
+            'coefficient': {'beta': float(last_line[7]), 'stdErr': float(last_line[8])},
+            'diff': {'beta': float(last_line[10]), 'stdErr': float(last_line[11])},
+            'pValue': float(last_line[6])
```

Comment:
Not sure if there's an easy alternative, but using indexes like this makes me nervous.  If a new column gets added some place other than the right most position to the source file then the indexes need to be updated here.  If there's no easy way to use column headers, then so be it.

Reply:
Yeah fair. I'll see. It is a TSV basically with a header, so I could zip the header and call by column name instead. Would at least catch when the headers change and explode.